spec/controllers/user_targets_controller_spec.rb

```diff
@@ -145,6 +145,26 @@
         end
       end
     end
+
+    context 'too many targets created' do
```

Comment:
Start the context with 'when', in this case 'when there are too many targets created'